exampleSite/content/about/pricing/plan-3.md

```diff
@@ -0,0 +1,22 @@
++++
+weight = 30
+
+title = "Enterprise plan"
+# subtitle = ""
+
+price = "Custom"
+# highlight = true
+
+button_text = "Contact us"
+link = "http://example.com"
++++
+
+**Basic** feature
```

Comment:
Gives a lot of flexibility on the text. My thinking was more in line with something like:
```
[[features]]
  text = "**Basic** feature
  icon = "fab fa-linkedin-in"
```

Your idea is definitely easier without considering the icon being added via css. Needs a bit more thinking I think.